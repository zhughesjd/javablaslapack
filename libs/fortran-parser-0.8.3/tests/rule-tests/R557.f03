! Testing common block statements
common i
common /name1/ i1, i2
common // i2
common i3 / / j
common /name1/ i4, /name2/ j1 /name3/ k, // l
end
