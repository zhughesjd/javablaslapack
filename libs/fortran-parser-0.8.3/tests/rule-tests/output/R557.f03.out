R1101:main-program__begin:
R 557:common-block-name: id=null
R 558:common-block-object-list__begin:
R 558:common-block-object: id=[@4,41:41='i',<206>,2:7] hasShapeSpecList=false
R 558:common-block-object-list: count=1
R 557:common-stmt: numBlocks=1
R 557:common-block-name: id=[@9,51:55='name1',<206>,3:8]
R 558:common-block-object-list__begin:
R 558:common-block-object: id=[@12,58:59='i1',<206>,3:15] hasShapeSpecList=false
R 558:common-block-object-list: count=1
R 557:common-stmt: numBlocks=1
R 557:common-block-name: id=null
R 558:common-block-object-list__begin:
R 558:common-block-object: id=[@20,72:73='i2',<206>,4:11] hasShapeSpecList=false
R 558:common-block-object-list: count=1
R 557:common-stmt: numBlocks=1
R 557:common-block-name: id=null
R 558:common-block-object-list__begin:
R 558:common-block-object: id=[@24,82:83='i3',<206>,5:7] hasShapeSpecList=false
R 558:common-block-object-list: count=1
R 557:common-block-name: id=null
R 558:common-block-object-list__begin:
R 558:common-block-object: id=[@30,89:89='j',<206>,5:14] hasShapeSpecList=false
R 558:common-block-object-list: count=1
R 557:common-stmt: numBlocks=2
R 557:common-block-name: id=[@35,99:103='name1',<206>,6:8]
R 558:common-block-object-list__begin:
R 558:common-block-object: id=[@38,106:107='i4',<206>,6:15] hasShapeSpecList=false
R 558:common-block-object-list: count=1
R 557:common-block-name: id=[@42,111:115='name2',<206>,6:20]
R 558:common-block-object-list__begin:
R 558:common-block-object: id=[@45,118:119='j1',<206>,6:27] hasShapeSpecList=false
R 558:common-block-object-list: count=1
R 557:common-block-name: id=[@48,122:126='name3',<206>,6:31]
R 558:common-block-object-list__begin:
R 558:common-block-object: id=[@51,129:129='k',<206>,6:38] hasShapeSpecList=false
R 558:common-block-object-list: count=1
R 557:common-block-name: id=null
R 558:common-block-object-list__begin:
R 558:common-block-object: id=[@58,136:136='l',<206>,6:45] hasShapeSpecList=false
R 558:common-block-object-list: count=1
R 557:common-stmt: numBlocks=4
R 204:specification-part: numUseStmts=0 numImportStmts=0 numDeclConstructs=5
R1103:end-program-stmt: id=null
R1101:main-program: hasProgramStmt=false hasExecutionPart=false hasInternalSubprogramPart=false
